src/main/java/uk/gov/ons/ctp/response/collection/exercise/service/EventService.java

```diff
@@ -20,7 +22,7 @@
     reminder3(false),
     ref_period_start(false),
     ref_period_end(false),
-    employment_date(false);
+    employment(false);
```

Comment:
Does this exist in any enums in the db? also will we need to update any events that have already been created?